Biodome/BiodomeTinkering.py

```diff
@@ -0,0 +1,90 @@
+import pygame, sys, math
+from pygame.locals import *
+
+pygame.init()
+pygame.mouse.set_visible(1)
+
+
+window_width = 1920
+window_height = 1080
+window_size = (window_width, window_height)
+
+screen = pygame.display.set_mode(window_size)
+bg = pygame.image.load("C:\Users\Kieran\Desktop\Tinkering\Biodome.jpg").convert_alpha()
+screen.fill((255,0,0))
+screen.blit(bg, (0, 0))
+
+PixelArray = pygame.PixelArray(screen)
+
+def purplesky():
+    for x in range(window_width):
+        for y in range(window_height):
+            R = screen.get_at((x,y)).r
+            G = screen.get_at((x,y)).g
+            B = screen.get_at((x,y)).b
+            if R > 230 and G > 230 and B > 230:
+                screen.set_at((x,y), (221,160,221))
+
+def greypicture():
+    for x in range(window_width):
+        for y in range(window_height):
+            R = screen.get_at((x,y)).r
+            G = screen.get_at((x,y)).g
+            B = screen.get_at((x,y)).b
+            Grey = (R + G + B)/3
+            PixelArray[x,y] = (Grey, Grey, Grey)
+
+def pinktree():
+    for x in range(window_width):
+        for y in range(window_height):
+            R = screen.get_at((x,y)).r
+            G = screen.get_at((x,y)).g
+            B = screen.get_at((x,y)).b
+            if R < 50 and G < 100 and B < 80:
+                screen.set_at((x,y), (238,130,238))
+
+def idkyet():

```

Comment:
Calls an "Out of Range" error when run. This is because of the "X - 1" phrase within your code. This means that the coordinates will be offscreen after the first iteration.